src/gurobi_logtools/parsers/crossover.py

```diff
@@ -0,0 +1,104 @@
+import re
+from typing import Dict
+
+from gurobi_logtools.parsers.util import (
+    Parser,
+    float_pattern,
+    typeconvert_groupdict,
+    ParseResult,
+)
+
+
+class CrossoverParser(Parser):
+    # The pattern indicating the initialization of the parser
+    crossover_start_pattern = re.compile(
+        r"Crossover log...",
```

Comment:
The crossover after barrier has a slightly different start pattern to crossover after pdhg.  See `tests/assets/mip.log` for example.  This will be an easy fix, but it will (and should) break some tests.